.github/workflows/ci.yml

```diff
@@ -2,6 +2,11 @@ name: CI
 
 on:
   push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
Do we really only want this on pull requests to the main branch?
I don't think we will have any on another branch, but there should be a setting that says "on every pull request", shouldn't it?

Reply:
i also thought that this might be too complicated, that's why the original ci.yml had only `push:` and nothing else. But then the ci steps where triggered twice for (forced) pushes to existing pull requests. I think this describes the problem quite well: https://github.com/orgs/community/discussions/26276
not sure if it's the best solution, but it should fix some problems.

Reply:
Ok, makes sense. If we ever have MRs to a different branch but main, we can change it anyways.